class_photo/bot.py

```diff
@@ -10,26 +10,34 @@
 bot = commands.Bot('-cp')
 
 def main():
-    load_dotenv()
     bot.run(os.getenv("TOKEN"))
```

Comment:
This statement should be under try and catch to handle the situation when `TOKEN` is not set accurately in the environment variable.